bin/middlewares/unsupported_browser.js

```diff
@@ -1,10 +1,24 @@
+function isOldIE(userAgent) {
+  return userAgent.match('Trident'); // Test for IE <= 11
+}
+
+function isOldiOS(userAgent) {
+  if (/iP(hone|od|ad)/.test(userAgent)) {
+    const v = userAgent.match(/OS (\d+)_(\d+)_?(\d+)?/);
+    if (v && v[1]) {
+      return parseInt(v[1], 10) < 10; // test if major iOS version is < 10
+    }
+  }
+}
+
 function isUnsupported(userAgent = '') {
-  return userAgent.match('Trident'); // old IE detection
+  if (isOldIE(userAgent) || isOldiOS(userAgent)) {
```

Comment:
I don`t get why you prefer writing
```js
if (expressionThatReturnsABool) {
   return true;
} 
// and once again, missing `return false;` here 
```
than just
```js
return expressionThatReturnsABool;
```

Reply:
sorry for the zig-zags, I still have issues to make choices between legibility and avoiding to write unnecessary code
for me the "return false" were unnecessary, and the "if(thing) return true" were much easier to read, but no prob, it's fixed :D